app/core/config.py

```
@@ -61,6 +89,25 @@ def RABBITMQ_URL(self) -> str:
         return f"amqp://{self.RABBITMQ_USERNAME}:{self.RABBITMQ_PASSWORD}" \
                f"@{self.RABBITMQ_HOST}:{self.RABBITMQ_PORT}/"
 
+    @computed_field
+    @property
+    def DATABASE_URL(self) -> str:
```

Pylint (3.3.7)

[convention] Attribute name "DATABASE_URL" doesn't conform to snake_case naming style

(C0103)